Appearances/00.lua

```diff
@@ -12,6 +12,7 @@ local CLOTH			= SetCollector.CLOTH
 local LEATHER		= SetCollector.LEATHER
 local MAIL			= SetCollector.MAIL
 local PLATE			= SetCollector.PLATE
+local ANYARMOR			= SetCollector.ANYARMOR
```

Comment:
Great idea. This will make it more legible. Can you change these new variables to include an underscore)?
```suggestion
local ANY_ARMOR			= SetCollector.ANY_ARMOR
```